base/src/org/adempiere/engine/StorageEngine.java

```diff
@@ -47,7 +47,7 @@
 	/**	Logger							*/
 	protected static transient CLogger	log = CLogger.getCLogger (StorageEngine.class);
 	
-	public static void createTrasaction (
+	public static void createTransaction(
```

Comment:
Please make this code more atomic, the refactory can be change in other pull request as Source code Improve